packages/backend/package.json

```diff
@@ -66,11 +66,11 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
-    "@types/luxon": "^3.0.0",
```

Comment:
It might be worthwhile to add a GitHub action to check for people hand editing `package.json` files.